demisto_sdk/commands/common/tools.py

```diff
@@ -1349,7 +1349,9 @@ def get_pack_ignore_content(pack_name: str) -> Union[ConfigParser, None]:
                 f"Error when retrieving the content of {_pack_ignore_file_path}\n{err=}"
             )
             return None
-    logger.error(f"{_pack_ignore_file_path} does not exist for pack {pack_name}")
+    logger.info(
```

Comment:
can use warn, i think it makes it orange

Reply:
originally it was `info`, still change it to warning?

Reply:
yes, imo